export { default as catchAsyncError } from "./catchAsyncError";
